Positional obs report formats

KingShark1 opened the pull request.

KingShark1 committed:
add equatorial to horizontal conversion

KingShark1 committed:
add time format string, remove test document

KingShark1 committed:
add time format string, remove test document

hornig commented on orbitdeterminator/.vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/kingshark/.virtualenvs/orbit_determinator/bin/python"
```
I think we do not need this ;).
please remove.

KingShark1 replied:
actually, i would need that on my machine, it isn't needed though.

PerezHz replied:
You can add the `.vscode` folder to `.gitignore`, so that git does not keep track of those files

PerezHz replied:
Unresolving this, since file is still tracked by git... You can un-track this file on git (e.g., `git rm --cached <FILE>`), and then add the following line:

```
.vscode/settings.json 
```

to `.gitignore`

hornig replied:
@PerezHz shall I just add it to .gitignore as you stated it?

PerezHz replied:
@hornig sure, I think it's sufficient to add `orbitdeterminator/.vscode`

hornig committed:
Merge pull request #233 from aerospaceresearch/Positional_Obs_Report_Formats

Positional obs report formats

hornig committed:
restructuring orbital_elements

restructuring orbital_elements

hornig committed:
adds better gibbs example

gibbs works best if 3 observations in one half orbit is taken.
the previous approach used the nieghboring points. this uses now the latest point in the first half orbit and the point on half index (not good implementation but works for example

hornig committed:
cleaning code

cleaning code

hornig committed:
deleted filtered.csv

filtered.csv is generated at every run.

KingShark1 committed:
Merge branch 'master' of https://github.com/KingShark1/orbitdeterminator into Positional_Obs_Report_Formats

hornig committed:
adds "none" to filters

now you can use the original data as well

hornig committed:
adds lamberts method for 3 observations

adds lamberts method for 3 observations. 2 is also okay, but for now 3 as for gibbs method, that really needs 3 observations

KingShark1 committed:
complete el-az to ra-dec conversion

KingShark1 committed:
add gauss method

hornig committed:
adds raw string to split statement

follows https://docs.python.domainunion.de/3/library/re.html
solves "DeprecationWarning: invalid escape sequence \s"

hornig committed:
Merge pull request #235 from aerospaceresearch/orbitdeterminator

Orbitdeterminator

KingShark1 committed:
Merge pull request #1 from aerospaceresearch/master

updating on 28th mar

KingShark1 committed:
Merge pull request #2 from KingShark1/master

some error

hornig commented:
stalled progress. closing it
